backend/data/src/main/kotlin/io/tolgee/service/query_builders/translationViewBuilder/QueryGlobalFiltering.kt

```diff
@@ -98,6 +98,14 @@ class QueryGlobalFiltering(
     }
   }
 
+  private fun getFilterNamespace(): List<String>? {
+    val filterNamespace = params.filterNamespace
+    if (filterNamespace != null && filterNamespace.isEmpty()) {
+      return listOf("")
```

Comment:
Will it work also in this case? `?filterNamespace=&filterNamespace=test`

Reply:
Yes.